### PR TITLE
Allow non admin people to access skillset

### PR DESCRIPTION
**Pull request by lkleisa**
[Issue 354](https://github.com/puzzle/skills/issues/354)
The skillset page should become read-only for all non-admin members. Meaning these non-admin members will be able to view the page but won't be able to edit anything.

### CODE REVIEW

**Comment by lkleisa** [comment]
I could copy mostly the code from [Old PR 441](https://github.com/puzzle/skills/pull/441). I added grayed out buttons: 
Non admin members have now the function to view the skillset page. They can view all the things. The create new skill and the edit button are grayed out for non admins and on click, there is an error message in a toastr pop up.  
Tests in frontend are adjusted. Currently, they are failing, because I added keyCloakService to frontend/app/components/skills-list-row-show.js. This service must be injected in the Test frontend/tests/integration/components/skills-list-row-show-test.js. I couldn't do it and had no help. 
Maybe, changes in backend are missing. In requirements, it says, that we have to make sure that both ends are covered and can't be accessed through some workaround.

**Comment by Robin481** [comment]
Looks good so far, goob job! We'll look at it together tomorrow :smile_cat:

File: frontend/app/templates/components/skills-list-row-show.hbs [hunk 1]
```diff
@@ -29,7 +29,7 @@
   {{if this.skill.portfolio this.skill.portfolio "-"}}
 </td>
 <td>
-  <a role="button" onclick={{this.startEditing}} class="edit-buttons skill-edit-button">
+  <a role="button" onclick={{this.startEditing}} class="edit-buttons skill-edit-button" style='{{unless this.isAdmin "-webkit-filter: grayscale(100%);"}}'>
```
Comment (Robin481) [inline frontend/app/templates/components/skills-list-row-show.hbs:32]: it probably makes more sense to add the css class conditionally than putting the style here. Furthermore I thinkg webkit filter is Firefox specific if i am not mistake, make sure to check that.

**Review by Robin481 — APPROVED** [review]
Nice :partying_face:

**Pull request merged**